### PR TITLE
Adding exceptions for self-signed SSL certificates in the Ceph Dashboard

### PR DESCRIPTION
**Pull request by votdev**
Signed-off-by: Volker Theile <vtheile@suse.com>

### CODE REVIEW

File: xml/bp_troubleshooting_dashboard.xml [hunk 1]
```diff
@@ -290,4 +290,19 @@ Error ENOENT: User <replaceable>USERNAME</replaceable> does not exist
...
+  <para>
+   The browser will show an error page when accessing embedded Grafana pages
+   that are using self-signed SSL certificates. Most browsers will not show the
+   button to add an exception for those pages. Instead, the frame has to be
+   opened in a separate tab to add an exception. In Firefox, for example, open
+   the context menu by clicking the right mouse button and select the menu
+   <emphasis>This Frame > Open Frame in New Tab</emphasis>. In the new opened
+   browser tab press the <emphasis>Advanced</emphasis> button followed by the
+   <emphasis>Accept the risk and continue</emphasis> button. Finally, reload
+   the &dashboard; page to see the embedded Grafana pages.
```
Comment (tbazant) [inline xml/bp_troubleshooting_dashboard.xml:305]:
```suggestion
   The Web browser will show an error page when accessing embedded &grafana; pages
   that are using self-signed SSL certificates. Most browsers will not show the
   button to add an exception for those pages. Instead, you need to open
   the frame in a separate browser tab to add an exception. In Firefox, for example, open
   the context menu by clicking the right mouse button and select the menu
   <menuchoice><guimenu>This Frame</guimenu> <guimenu>Open Frame in New Tab</guimenu</menuchoice>. In the new opened
   browser tab, press the <guimenu>Advanced</guimenu> button followed by the
   <guimenu>Accept the risk and continue</guimenu> button. Finally, reload
   the &dashboard; page to see the embedded &grafana; pages.
```

**Review by tbazant — CHANGES_REQUESTED** [review]
Small update.

**Comment by tbazant** [comment]
@votdev BTW should this be backported to SES 7 as well? If yes, add `backport to SES7` label please.

File: xml/bp_troubleshooting_dashboard.xml [hunk 2]
```diff
@@ -290,4 +290,42 @@ Error ENOENT: User <replaceable>USERNAME</replaceable> does not exist
...
+  <title>Adding exceptions for self-signed SSL certificates in the &dashboard;</title>
+
+  <para>
+   The Web browser shows an error page when accessing any embedded &grafana;
+   pages using self-signed SSL certificates. Most browsers do not display the
+   button to add an exception for those pages.
+  </para>
+
+  <para>
+   Follow the next steps to enable add the exception.
```
Comment (dariavladykina) [inline xml/bp_troubleshooting_dashboard.xml:303]:
```suggestion
   Follow the next steps to add the exception.
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please take a look at a minor change.

Thanks,
Daria

**Review by tbazant — APPROVED** [review]
LGTM

**Pull request merged**